osu.Server.DifficultyCalculator/Commands/AllCommand.cs

```diff
@@ -14,13 +14,21 @@ public class AllCommand : CalculatorCommand
         [Option(CommandOptionType.NoValue, Template = "-r|--ranked", Description = "Only calculate difficulty for ranked/approved/qualified/loved maps.")]
         public bool RankedOnly { get; set; }
 
+        [Option("--sql", Description = "Specify a custom query to limit the scope of beatmaps")]
+        public string? CustomQuery { get; set; }
+
+        [Option("--from", Description = "The minimum beatmap id to calculate the difficulty for.")]
+        public int StartId { get; set; }
+
         protected override IEnumerable<int> GetBeatmaps()
         {
             using (var conn = DatabaseAccess.GetConnection())
             {
                 var condition = CombineSqlConditions(
                     RankedOnly ? "`approved` >= 1" : null,
-                    "`deleted_at` IS NULL"
+                    $"`beatmap_id` >= {StartId}",
```

Comment:
works in my head even if parameter is omitted because then the default value of 0 is used thus this clause becomes `beatmap_id >= 0` which should catch everything that's relevant